libs/transactions/src/lib/services/transactions-journey-config.service.ts

```diff
@@ -1,8 +1,9 @@
 import { Injectable, TemplateRef } from '@angular/core';
+import { AdditionalDetailsContext } from '../directives/transaction-additional-details.directive';
 
 @Injectable()
 export class TransactionsJourneyConfiguration {
   pageSize = 20;
   slimMode = true;
-  additionalDetailsTpl: TemplateRef<any> | undefined = undefined;
+  additionalDetailsTpl: TemplateRef<AdditionalDetailsContext> | undefined = undefined;
```

Comment:
This is all an engineer would need to do to override the default null template, right?

Reply:
The developer should define the interface for the context in the directive, and write a guard for it

And they should import this (like here) in the config service to correctly type the templates stored there